README.md

```diff
@@ -20,3 +20,17 @@ docker build -t ibc-explorer .
 
 docker run -e API_URL=tcp://host.docker.internal:8080 -p 5001:5000 ibc-explorer
 ```
+
+## E2E Testing
+To run tests on a development server using the Cypress GUI:
+```shell
+npm run test
```

Comment:
I saw this error when I tried to run `npm run test` --> sh: start-server-and-test: command not found

Is that expected?

Reply:
I think just needed cypress installed